docker/ubuntu-full/Dockerfile

```diff
@@ -483,6 +501,31 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     && apt-get install -y -V libparquet${ARROW_SOVERSION} \
     && apt-get install -y -V libarrow-dataset${ARROW_SOVERSION}
 
+# Manually install ADBC packages from Ubuntu 22.04 as there are no 24.04 packages at time of writing.
+RUN if test "$(uname -p)" = "x86_64"; then \
+    wget -q https://apache.jfrog.io/artifactory/arrow/ubuntu/pool/jammy/main/a/apache-arrow-adbc/libadbc-driver-manager102_14-1_amd64.deb \
+    && wget -q https://apache.jfrog.io/artifactory/arrow/ubuntu/pool/jammy/main/a/apache-arrow-adbc/libadbc-driver-manager-dev_14-1_amd64.deb \
+    && wget -q https://apache.jfrog.io/artifactory/arrow/ubuntu/pool/jammy/main/a/apache-arrow-adbc/libadbc-driver-sqlite102_14-1_amd64.deb \
+    && wget -q https://apache.jfrog.io/artifactory/arrow/ubuntu/pool/jammy/main/a/apache-arrow-adbc/libadbc-driver-sqlite-dev_14-1_amd64.deb \
+    && wget -q https://apache.jfrog.io/artifactory/arrow/ubuntu/pool/jammy/main/a/apache-arrow-adbc/libadbc-driver-snowflake102_14-1_amd64.deb \
+    && wget -q https://apache.jfrog.io/artifactory/arrow/ubuntu/pool/jammy/main/a/apache-arrow-adbc/libadbc-driver-snowflake-dev_14-1_amd64.deb \
+    && dpkg -i libadbc-driver-manager102_14-1_amd64.deb \
+    && dpkg -i libadbc-driver-manager-dev_14-1_amd64.deb \
+    && dpkg -i libadbc-driver-sqlite102_14-1_amd64.deb \
+    && dpkg -i libadbc-driver-sqlite-dev_14-1_amd64.deb \
+    && dpkg -i libadbc-driver-snowflake102_14-1_amd64.deb \
+    && dpkg -i libadbc-driver-snowflake-dev_14-1_amd64.deb \
+    && rm -f libadbc*.deb; \
+    fi
+
+# Install libduckdb
+RUN if test "$(uname -p)" = "x86_64"; then \
+    wget -q https://github.com/duckdb/duckdb/releases/download/v1.1.1/libduckdb-linux-amd64.zip \
```

Comment:
Could the `wget -q` commands be changed to `curl -LO -fsS` so the build fails with an error message on HTTP errors?

Reply:
done, thanks